content/_code-samples/use-tickets/js/use-tickets-multisig.js

```diff
@@ -0,0 +1,129 @@
+
+    // Signer keys don't need to be funded on the ledger, it only needs to be cryptographically valid
+    // Thus, we could generate keys and set them as signers without the need to fund their accounts
+    // But we'll still fund them for testing purposes...
+    const { wallet: wallet_1 } = await client.fundWallet()
+    const { wallet: wallet_2 } = await client.fundWallet()
+    const { wallet: wallet_3 } = await client.fundWallet()
+
+    console.log(" Main Account: ", main_wallet.address)
+    console.log("         Seed: ", main_wallet.seed)
+
+    console.log("\n Signer 1: ", wallet_1.address)
+    console.log(" Signer 2: ", wallet_2.address)
+    console.log(" Signer 3: ", wallet_3.address)
+
+    // Send SignerListSet transaction
+    // Since each signer is given a signer weight of 1 and there are 3 signers, the maximum quorom would be 3
+    // SignerQuorom is a target number for the signer weights
+    // A multisig from this list is valid only if the sum weights of the signatures provided is greater than or equal to the SignerQuorom
+    const signerLiSetSignerList_tx = {
+        TransactionType: "SignerListSet",
+        Account: main_wallet.classicAddress,
+        SignerEntries: [
+          {
+            SignerEntry: {
+              Account: wallet_1.classicAddress,
+              SignerWeight: 1,
+            },
+          },
+          {
+            SignerEntry: {
+              Account: wallet_2.classicAddress,
+              SignerWeight: 1,
+            },
+          },
+          {
+            SignerEntry: {
+              Account: wallet_3.classicAddress,
+              SignerWeight: 1,
+            },
+          }
+        ],
+        SignerQuorum: 2,
+      }
+
+    const signerLiSetSignerList_tx_prepared = await client.autofill(signerLiSetSignerList_tx)
+    const SetSignerList_tx_signed = main_wallet.sign(signerLiSetSignerList_tx_prepared)
+    console.log("\n SignerListSet Tx hash:", SetSignerList_tx_signed.hash)
+  
+    const setsignerlist_submit = await client.submitAndWait(SetSignerList_tx_signed.tx_blob)
+    console.log("           Submit result:", setsignerlist_submit.result.meta.TransactionResult)
+    
+    const CreateTicket_tx = await client.autofill({
+      "TransactionType": "TicketCreate",
+      "Account": main_wallet.address,
+      "TicketCount": 3
+    })
+  
+    const CreateTicket_tx_signed = main_wallet.sign(CreateTicket_tx)
+    console.log("\n CreateTicket Tx hash:", CreateTicket_tx_signed.hash)
+  
+    const ticket_submit = await client.submitAndWait(CreateTicket_tx_signed.tx_blob)
+    console.log("        Submit result:", ticket_submit.result.meta.TransactionResult)
+    
+    const ticket_response = await client.request({
+        "command": "account_objects",
+        "account": main_wallet.address,
+        "ledger_index": "validated",
+        "type": "ticket"
+    })
+
+    console.log(`\n- Tickets issued by ${main_wallet.address}:\n`)
```

Comment:
Oh, looks like you already did use a template literal. Let's make it consistent for all other log statements :)